### PR TITLE
Add OdiseoTeam SyliusVendorPlugin Recipe

### PR DESCRIPTION
**Pull request by bigboss86**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: odiseoteam/sylius-vendor-plugin/1.2/manifest.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "bundles": {
+        "Vich\\UploaderBundle\\VichUploaderBundle": ["all"],
```
Comment (Nyholm) [inline odiseoteam/sylius-vendor-plugin/1.2/manifest.json:3]: This is not needed.

File: odiseoteam/sylius-vendor-plugin/1.2/config/packages/odiseo_sylius_vendor_plugin.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+imports:
+    - { resource: "@OdiseoSyliusVendorPlugin/Resources/config/config.yaml" }
```
Comment (Nyholm) [inline odiseoteam/sylius-vendor-plugin/1.2/config/packages/odiseo_sylius_vendor_plugin.yaml:2]: In this file you keep config to `knp_gaufrette` , `vich_uploader` and `liip_imagine`. Some of that config will override the any existing config and cause issues really hard to find. 

Im not a big fan of solutions like this.

**Review by Nyholm — CHANGES_REQUESTED** [review]
Im not a fan of Sylius plugins like this... 

I fear this is a cause of hard to debug issues.

File: odiseoteam/sylius-vendor-plugin/1.2/config/packages/odiseo_sylius_vendor_plugin.yaml (hunk 2 above)
Comment (bigboss86) [reply odiseoteam/sylius-vendor-plugin/1.2/config/packages/odiseo_sylius_vendor_plugin.yaml:2]: @Nyholm We are only defining the configuration that we are going to use in this plugin, which will merge with the configuration of each project.
What solution do you think is correct?

File: odiseoteam/sylius-vendor-plugin/1.2/manifest.json (hunk 1 above)
Comment (bigboss86) [reply odiseoteam/sylius-vendor-plugin/1.2/manifest.json]: @Nyholm Removed. Thanks!

**Review by bigboss86 — COMMENTED** [review]
I have left you the revision changes of the recipe

**Comment by bigboss86** [comment]
@Nyholm could you merge this PR?

**Review by Nyholm — APPROVED** [review]
I removed version 1.3 and 1.4 as they are identical to 1.2

**Comment by bigboss86** [comment]
Thanks!

**Pull request merged**